runtime/src/system_instruction_processor.rs

```diff
@@ -666,21 +677,21 @@ mod tests {
     fn test_create_with_zero_lamports() {
         // create account with zero lamports transferred
         let new_owner = Pubkey::new(&[9; 32]);
-        let from = solana_sdk::pubkey::new_rand();
-        let from_account = AccountSharedData::new_ref(100, 1, &solana_sdk::pubkey::new_rand()); // not from system account
+        let from = Pubkey::new_unique();
+        let from_account = AccountSharedData::new_ref(100, 0, &Pubkey::new_unique()); // not from system account
 
-        let to = solana_sdk::pubkey::new_rand();
+        let to = Pubkey::new_unique();
         let to_account = AccountSharedData::new_ref(0, 0, &Pubkey::default());
 
         assert_eq!(
             create_account(
-                &KeyedAccount::new(&from, false, &from_account), // no signer
-                &KeyedAccount::new(&to, false, &to_account),
+                &KeyedAccount::new(&from, true, &from_account),
+                &KeyedAccount::new(&to, true, &to_account),
```

Comment:
doesn't need to be a signer

Reply:
oops, I thought this was just a transfer, never mind

Reply:
I literally spent about an hour making sure that it does need to be a signer :sweat_smile:  I was worried that it was possible to create an ephemeral account without signing for it.  The tests are just written a bit strangely, so I wanted to clarify them